web/js/newberry.js

```diff
@@ -3085,7 +3085,16 @@ function splitPage(event, tool) {
             $("#compareSplit").css("width", splitWidthAdjustment);
             break
         case "partialTrans":
-            var currentCanvasLabel = transcriptionFolios[currentFolio - 1]["label"];
+            //var currentCanvasLabel = transcriptionFolios[currentFolio - 1]["label"];
+            var currentCanvasID = transcriptionFolios[currentFolio - 1]["@id"];
+            var utlID = "";
+            //We need to get the UTL canvas id for this particular canvas to support direct linking to the transcription for this object
+            utlID = currentCanvasID.substring(currentCanvasID.lastIndexOf("paleography:"), currentCanvasID.length - 1).replace("paleography:", "");
```

Comment:
I thought we wanted the first Index, it is just the case that for now we only have one match. Also, you may want a regex pattern for the match so we get only what we are expecting if there is a weird change.

At any rate, I think the `.replace` is excessive and you may just want `lastIndexOf()+12` instead